navigation/FeedNavigator.tsx

```diff
@@ -22,7 +25,11 @@ const FeedNavigator: React.FC = () => {
         options={({ navigation }) => ({
           headerTitle: 'Feed',
           // eslint-disable-next-line react/display-name
-          headerRight: () => <HeaderIcon name="bell" onPress={() => navigation.navigate('Notifications')} />,
+          headerRight: () => (
+            <BadgeIcon value={count} onPress={() => navigation.navigate('Notifications')}>
+              <HeaderIcon name="bell" />
```

Comment:
HeaderIcons in other pages should also have the same padding (so that they at the same position in all headers)

Reply:
I changed all header icons to be the same